services-manager/src/main/resources/application-services.yaml

```diff
@@ -20,23 +20,21 @@
 app:
   version: '@project.version@'
 
-eureka:
-  client:
-    enabled: true
-
 manager:
   # Defaults used internally to the yaml ONLY - these are not interpreted by any java application, just to minimise duplication of long filepaths etc.
   default:
     # Library includes relative to manager.root above (e.g. ../palisade-examples => palisade-services/../palisade-examples)
     # While the example library happens to be the same version as services, it should be declared here as a static name
-    library: ../palisade-examples/example-library/target/
+    example-library: ../Palisade-examples/example-library/target/example-library-0.4.0-SNAPSHOT.jar
```

Comment:
any way to parameterise the versions here?

Reply:
See comment just above, yes but there is no guarantee one version matches another.
It could be wildcarded for now?